docs/caveat-unique-values.doctest.rst.inc

```diff
@@ -0,0 +1,121 @@
+.. _caveat-unique-values:
+
+Values Must Be Unique
```

Comment:
I don't think this section belongs under Caveats any more. This is actually a strength, a useful guarantee. Some of this section could come under Basic Usage.